bitnami/spark/templates/statefulset-master.yaml

```diff
@@ -71,28 +71,27 @@ spec:
             - name: SPARK_MASTER_OPTS
               value: {{ .Values.master.configOptions | quote }}
             {{- end }}
-            {{- if eq ( .Values.security.rpc.authenticationEnabled | quote ) "yes" }}
+            {{- if .Values.security.rpc.authenticationEnabled }}
             - name: SPARK_RPC_AUTHENTICATION_ENABLED
               value: {{ .Values.security.rpc.authenticationEnabled | quote }}
             - name: SPARK_RPC_AUTHENTICATION_SECRET
               valueFrom:
                 secretKeyRef:
-                {{- if .Values.security.passwordsSecretName }}
+                  {{- if .Values.security.passwordsSecretName }}
                   name: {{ .Values.security.passwordsSecretName }}
-                {{- else }}
+                  {{- else }}
                   name: {{ include "spark.fullname" . }}-secret
-                {{- end }}
                   key: rpc-authentication-secret
             {{- end }}
-            {{- if eq ( .Values.security.encryptionEnabled | quote ) "yes" }}
+            {{- if .Values.security.rpc.encryptionEnabled }}
             - name: SPARK_RPC_ENCRYPTION_ENABLED
               value: {{ .Values.security.rpc.encryptionEnabled | quote }}
             {{- end }}
-            {{- if eq ( .Values.security.storageEncriptionEnabled | quote ) "yes" }}
```

Comment:
I think the fix to the storage issue you are trying to fix is to correct this typo.